client/app/assets/less/redash/redash-newstyle.less

```diff
@@ -804,6 +804,48 @@ text.slicetext {
     text-align: center !important;
   }
 
+  dashboard-page {
+    .dashboard-header {
+      padding: 0 !important;
+
+      .page-title h3 {
+        margin-bottom: 0 !important;
+        font-size: 18px;
+        line-height: 2;
+      }
+
+      .dashboard__control {
+        margin: 5px 0;
+      }
+    }
+
+    .btn-fullscreen, .btn-share, .btn-more, tags-control, .btn-refresh-dropdown {
+      display: none;
```

Comment:
You don't need specific classes for these buttons just to hide them. You can use the `hidden-xs` class instead.

Reply:
Yes, but I'm trying to avoid those as it might lead to more complications on the long run (we had some issues with these in the past). Not sure which one makes more sense to be honest and I think we don't have a consistent pattern here. Happy to change if you think we should do so.

Reply:
What kind of complications?

Reply:
If I remember correctly it was with flexboxes when we originally used different flex classes to control the flexbox but then we had to change things on different screen size. So we had to introduce specific classes instead and use media queries accordingly to control how it changes.

Not sure if I'm explaining it correctly.

Anyway, translating to this specific case: what if we need to change something else on this button in the future. Then we got to keep adding extra classes or just have 1 class and do the rest in Sass.

Reply:
_If_ we will need to change a button in a specific way, we can introduce an extra class. I don't see why should we add it when it's not needed though.

Also somewhat related: I want to avoid having unique snowflakes in our styling as much as possible. Of course it's not always possible, but we should aim there. So if we need to change a button, we should think whether it's a new style of general type of button.

Reply:
Alright, I'll change it accordingly!